src/main/java/uk/gov/companieshouse/confirmationstatementapi/model/json/ConfirmationStatementSubmissionJson.java

```diff
@@ -9,6 +9,9 @@
     @JsonProperty
     private String id;
 
+    @JsonProperty
```

Comment:
I think i can see what's happening now, we should be able to have just the 1 model instead of having a dao version and a json version. Should be able to use a mixture of annotations on it for json parsing and document persisting.

Reply:
just looking at objections api, i notice we had DTOs therem maybe that approach might reduce the class count?

Reply:
This is starting to sound like another discussion.

Reply:
actually, i'm not 100% sure about the 1 model vs 2 approach, perhaps get input from @ICoxCH and @sroberts2

Reply:
No this is no good - we need a discussion on teams. I think we're doing this the wrong way flooding the PR with comments like this.